crucible-mir/src/Mir/Trans.hs

```diff
@@ -359,6 +391,26 @@ staticPlace did = do
             MirPlace (G.globalType gv) <$> globalMirRef gv <*> pure NoMeta
         Nothing -> mirFail $ "cannot find static variable " ++ fmt did
 
+-- variant of staticPlace for slices
+-- tpr is the element type; len is the length
+staticPlaces :: HasCallStack => C.TypeRepr tp -> Int -> M.DefId -> MirGenerator h s ret (MirPlace s)
+staticPlaces tpr len did = do
+    sm <- use $ cs.staticMap
+    case Map.lookup did sm of
+        Just (StaticVar gv) -> do
```

Comment:
Would it be possible / reasonable for this to call `staticPlace` instead of duplicating the lookup code?  It looks like the `MirPlace` returned by `staticPlace` contains all the pieces you need

Reply:
It could, but it seems the wrong way to abstract things (these are two different ways to look up a static and construct a Place for it, and having the more complex one call the simpler one and then munge the results doesn't seem right)

So... I'd rather not I think

Reply:
FWIW I think one interpretation of `staticSlicePlace` is that it obtains a reference to the array `did` (e.g. `&[u8; 10]`), which is exactly the output of `staticPlace`, and then converts that array reference to a slice reference (`&[u8]`).  But implementation-wise I think it's fine either way.

Reply:
That is also true. hrm...

I guess at this point the path of least resistance is to not make more changes...